GTG/backends/unmaintained/backend_rtm.py

```diff
@@ -562,7 +562,7 @@ def _authenticate(self):
             if not self.token:
                 self.rtm = createRTM(
                     self.PUBLIC_KEY, self.PRIVATE_KEY, self.token)
-                subprocess.Popen(['xdg-open', self.rtm.getAuthURL()])
+                webbrowser.open(self.rtm.getAuthURL())
```

Comment:
Use Gtk.show_uri instead?

This is just a comment on this bit, not a full review

Reply:
This should be `Gio.AppInfo.launch_default_for_uri()` ([link](https://lazka.github.io/pgi-docs/#Gio-2.0/classes/AppInfo.html#Gio.AppInfo.launch_default_for_uri)) so it uses the right portal in the flatpak version.

Reply:
Used `Gio.AppInfo.launch_default_for_uri()`